### DISCUSSION TITLE
GitHub Rest API giving an incorrect updated_at field value

### DISCUSSION THREAD

**Discussion by aishwaryt123** [community/community#60016] [Apps, API and Webhooks]
### GitHub Rest API giving an incorrect updated_at field value.

Bug

### Body

I am making a request on Github API for repo details and it is returning an old updated_at field.

When I go to UI and check for the repo https://github.com/CloudHealth/cht-terraform-classic-prod,the last commit happened on 14th April 2023.
But when I call the endpoint https://api.github.com/user/repos?per_page=100&page=4 with a proper access token I am getting the updated_at field as "2022-02-11T05:34:33Z"  which is more than a year ago but the pushed_at field is coming as "2023-06-30T12:01:34Z".

Why updated_at field is not getting updated?

**Comment by project-orcon** [comment]
I'm having the same issue

**Reply by project-orcon** [reply]
oh the problem was that i wanted pushed_at not updated_at
